### ISSUE TITLE
Simplify fetching coverage grunt task  

### ISSUE THREAD

**Issue by MaciejBaj** [LiskHQ/lisk#658]
as environment variables are set by CI tool (jenkins) it might be not clear wchich of them should be set before running `fetchCoverage` grunt task

### Expected behaviour
`$HOST` variable should be set if it's empty

### Current behaviour
without `$HOST` variable `fetchCoverage` task crushes with error message:
```
>> curl: (3) <url> malf
>> ormed
>> Exited with code: 3.
>> Error executing child process: Error: Process exited with code 3.
Warning: Task "exec:fetchCoverage" failed. Use --force to continue.
```

**Comment by Isabello** [comment]
https://github.com/LiskHQ/lisk/blob/development/Jenkinsfile#L342

??

**Comment by MaciejBaj** [comment]
yes, on jenkins everything works fine. Task addresses using the task locally

**Comment by Isabello** [comment]
Yes ok. Can we do away with needing to set it at all?

**Comment by MaciejBaj** [comment]
yes ofc, but I would just leave it

**Comment by Isabello** [comment]
Closed by 7676c39
